### DISCUSSION TITLE
Want Flash .BIN for flash for starting

### DISCUSSION THREAD

**Discussion by lepetitchris** [josephdadams/TallyArbiter#554] [Q&A] [answered]
HI sorry but, i'm a very green on That,
i'just want starting with my 2 of : M5 Stick and M5 USB... it not flashing
But i try to flash  any of 2 with the .bin, is look like too old..
i'dn find other stuff.
i'm a 3.0.4

**Comment by josephdadams** [comment, marked as answer]
Yeah the 2.x listeners don’t work with 3.x server

**Reply by NutOne718** [reply]
I am having the same issue. Moved from 2.x to 3.x and my listeners, m5stick C+, don't show up under Connected Listeners. I tried flashing them with https://web.esphome.io/ because the ESPHome Flasher is depreciated. It did not work using the prebuild .BIN from here. What should I do?

**Reply by josephdadams** [reply]
And you then flashed the 3.0 version listeners?

**Reply by NutOne718** [reply]
Yes. I flashed it with the BIN from [here](https://github.com/josephdadams/TallyArbiter/actions/runs/8065048697) for the M5Stick C+

**Reply by wrldtvlr** [reply]
Pretty sure that's a v2 client. The v3 clients are included with the server download.

**Reply by NutOne718** [reply]
OH WOW!!!! I just downloaded the server and got the ino files from there and used IDE to flash it and BAAAAAM!!!!!! Thatnk you so much for the help guys!!!! @wrldtvlr @josephdadams
